common/schema/go2proto.to.go

```diff
@@ -86,14 +174,22 @@ func AWSIAMAuthDatabaseConnectorFromProto(v *destpb.AWSIAMAuthDatabaseConnector)
 	}
 
 	out = &AWSIAMAuthDatabaseConnector{}
-	if fieldPos, err := PositionFromProto(v.Pos); err != nil {
+	out.Pos, err = orZeroR(toResult(PositionFromProto(v.Pos))).result()
+	if err != nil {
 		return nil, fmt.Errorf("Pos: %w", err)
-	} else {
-		out.Pos = fromPtr(fieldPos)
 	}
-	out.Username = string(v.Username)
-	out.Endpoint = string(v.Endpoint)
-	out.Database = string(v.Database)
+	out.Username, err = orZeroR(toResult(ptr(string(v.Username)), nil)).result()
+	if err != nil {
+		return nil, fmt.Errorf("Username: %w", err)
+	}
+	out.Endpoint, err = orZeroR(toResult(ptr(string(v.Endpoint)), nil)).result()
+	if err != nil {
+		return nil, fmt.Errorf("Endpoint: %w", err)
+	}
```

Comment:
These could be collapsed a bit more:
```suggestion
	if out.Endpoint, err = orZeroR(toResult(ptr(string(v.Endpoint)), nil)).result(); err != nil {
		return nil, fmt.Errorf("Endpoint: %w", err)
	}
```